script/gen_fuzz.py

```diff
@@ -0,0 +1,180 @@
+#!/usr/bin/env python3
```

Comment:
This file is lacking a single line of comment.
Even a blank line between the hunge strings would be of help.

Maybe you could at least add a general description to the top of this.

Reply:
I have added comments